dask_ml/_partial.py

```diff
@@ -22,6 +23,13 @@ class _WritableDoc(ABCMeta):
     # TODO: Py2: remove all this
 
 
+_partial_deprecation = (
+    "'{cls.__name__}' is deprecated. Use "
+    "'dask_ml.wrappers.Incremental({base.__name__}(), **kwargs)' "
```

Comment:
Should `Incremental` be top-level?  This might be a broader discussion though about namespaces.

Reply:
I dislike the `.wrappers` namespace, but I haven't put much thought into a replacement.